### DISCUSSION TITLE
分享你们用 NovelAI Bot 制作的图片！

### DISCUSSION THREAD

**Discussion by shigma** [koishijs/novelai-bot#88] [Show and tell]
欢迎大家在这里分享你们使用 NovelAI Bot 绘制的图片！优秀的图片还可以加入 readme 让更多人看到哦！

注：请不要评论 NSFW 内容，否则会被我删除并且上传者可能遭 GitHub 封禁。

**Comment by xiaoyangdian** [comment]
怎么么看到分享的图片呢

**Comment by swanmsg** [comment]
配置完成后，出现“发生未知错误 (400)。”，如何解决？
